profiles/common/modules/custom/nexteuropa_varnish/nexteuropa_varnish.module

```diff
@@ -279,6 +279,20 @@ function nexteuropa_varnish_node_delete($node) {
   }
 }
 
+/**
+ * Implements hook_file_delete().
+ */
+function nexteuropa_varnish_file_delete($file) {
+  $scheme = file_uri_scheme($file->uri);
+  if ($scheme && file_stream_wrapper_valid_scheme($scheme)) {
```

Comment:
```suggestion
  if ($scheme !== FALSE && file_stream_wrapper_valid_scheme($scheme) === TRUE) {
```

Reply:
Let's give people the opportunity to learn the Drupal API just by reading simple code :-)